packages/scripts/src/commands/e2e-data/parse.ts

```diff
@@ -0,0 +1,92 @@
+import { Command } from "commander";
+import fs from "fs-extra";
+import path from "path";
+import XLSX from "xlsx";
+import prettier from "prettier";
+import * as mappings from "./mappings";
+import snippets from "./snippets";
+import { ROOT_DIR } from "../../paths";
+
+interface IProgramOptions {
+  input: string;
+  output: string;
+}
+/** [WIP] Helps turn XLSX data into cypress compatible test runs. */
+const program = new Command("parse");
+export default program
+  .description("Parse worksheet into JS!")
+  .requiredOption("-i --input <string>", "path to input xlsx file (relative to project root)")
+  .requiredOption("-o --output <string>", "path to output spec file (relative to project root")
+  .action((options: IProgramOptions) => {
+    // Setup Input and Outputs
+    const { input, output } = options;
+    const inputPath = path.resolve(ROOT_DIR, input);
+    const outputPath = path.resolve(ROOT_DIR, output);
+    fs.ensureDirSync(path.dirname(outputPath));
+    const workbook = XLSX.readFile(inputPath);
+    const outputs = ["/** This file has been automatically generated. Do not modify directly */"];
+
+    // Convert XLSX to JSON
+    const worksheets = ["Run", "Specify", "Describe"].reduce((cargo, v) => {
```

Comment:
At first I was confused why we weren't using the existing xlsx conversion but actually it makes a lot of sense for this to be handled independently right now given that we want to link data across the different sheets and create individual (not merged/collated) outputs. It also means development on the e2e feature won't be blocked by limitations with the existing parsers, so definitely good to keep for the short term at least